spec/vector_spec.rb

```diff
@@ -1392,11 +1394,14 @@
     let(:dv) { Daru::Vector.new [1, nil, 3, :a, Float::NAN, nil, Float::NAN, 1],
       index: 11..18 }
     context 'reject only nils' do
-      subject { dv.reject_values nil }
+      # subject { dv.reject_values nil }
       
-      it { is_expected.to be_a Daru::Vector }
-      its(:to_a) { is_expected.to eq [1, 3, :a, Float::NAN, Float::NAN, 1] }
-      its(:'index.to_a') { is_expected.to eq [11, 13, 14, 15, 17, 18] }
+      # it { is_expected.to be_a Daru::Vector }
+      # its(:to_a) { is_expected.to eq [1, 3, :a, Float::NAN, Float::NAN, 1] }
+      # its(:'index.to_a') { is_expected.to eq [11, 13, 14, 15, 17, 18] }
+      include_context 'reject values checker',
+        [dv, [nil]] => [[1, 3, :a, Float::NAN, Float::NAN, 1],
+          [11, 13, 14, 15, 17, 18]]
     end
```

Comment:
@zverok This is driving me nuts. I'm trying to make this work but it isn't. It gives error something like `'dv' is not available on an example group`. Could you please have a look?

Another thing I'm thinking is that this code doesn't look readable at all. Passing the expected index, along with expected values of the vector and also passing along the type makes it hard to read what's going on. What do you think?